scripts/linux/ci/install-libvirt.sh

```diff
@@ -17,9 +17,16 @@ apt-get install -y --no-install-recommends \
     libvirt-dev \
     libxml2-dev \
     libxslt1-dev \
+    nfs-kernel-server \
     qemu-kvm \
     qemu-utils \
     qemu \
     rsync \
     ruby-dev \
     zlib1g-dev
+
+modprobe -a kvm
+
+# This doesn't have effect if you don't open a new shell
```

Comment:
if this is run as root... I am afraid it will not add the correct user to libvirt and kvm group?

Reply:
Right. We have a few options: https://stackoverflow.com/questions/1629605/getting-user-inside-shell-script-when-running-with-sudo

Reply:
putting this back where is it was because it depends too much on the environment. If we move it here, we've to take into account many different situations/platforms.